test/test_torch.py

```diff
@@ -15389,6 +15389,88 @@ def test_minmax(self, device, dtype):
         self._test_minmax_helper(lambda x: torch._min_max(x)[0], np.min, device, dtype, skip_indices=True)
         self._test_minmax_helper(lambda x: torch._min_max(x)[1], np.max, device, dtype, skip_indices=True)
 
+    @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
+    @dtypes(*(torch.testing.get_all_int_dtypes() + [torch.bool]))
+    def test_maximum_minimum_int_and_bool(self, device, dtype):
+        ops = ((torch.maximum, np.maximum), (torch.minimum, np.minimum))
+        a_vals = (2, 3, 4, -2)
+        b_vals = (-1, 5, 2, 0)
+
+        for torch_op, numpy_op in ops:
+            a_tensor = torch.tensor(a_vals, device=device, dtype=dtype)
+            b_tensor = torch.tensor(b_vals, device=device, dtype=dtype)
+            tensor_result = torch_op(a_tensor, b_tensor)
+
+            out = torch.empty_like(a_tensor)
+            torch_op(a_tensor, b_tensor, out=out)
+
+            a_np = np.array(a_vals, dtype=torch_to_numpy_dtype_dict[dtype])
+            b_np = np.array(b_vals, dtype=torch_to_numpy_dtype_dict[dtype])
+            numpy_result = numpy_op(a_np, b_np)
+
+            self.assertEqual(tensor_result.cpu(), torch.from_numpy(numpy_result))
```

Comment:
I think you can compare directly to the NumPy array without converting it to a `torch.Tensor`.

Reply:
Addressed.